IdentityCore/src/cache/MSIDMacKeychainTokenCache.h

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import <Foundation/Foundation.h>
+#import "MSIDTokenCacheDataSource.h"
+
+@interface MSIDMacKeychainTokenCache : NSObject <MSIDTokenCacheDataSource>
+
+/*!
+ The name of the group to be used by default when creating an instance of MSIDKeychainTokenCache,
+ the default value is com.microsoft.identity.universalstorage.
+
+ If set to 'nil' the main bundle's identifier will be used instead.
+
+ Because the keychain usage is different on macOS than on iOS, note that this group is used
+ for keychain query filtering, not as an actual keychain group.
+
+ NOTE: Once an authentication context has been created with the default keychain
+ group, or +[MSIDMacKeychainTokenCache defaultKeychainCache] has been called, then
+ this value cannot be changed. Doing so will throw an exception.
+ */
+@property (class, nullable) NSString *defaultKeychainGroup;
+
+/*!
+ Default cache. Will be initialized with defaultKeychainGroup.
+ */
+@property (class, readonly, nonnull) MSIDMacKeychainTokenCache *defaultKeychainCache;
+
+/*!
+ Actual keychain sharing group used for queries.
+ May contain team id (<team id>.<keychain group>)
+ */
+@property (readonly, nonnull) NSString *keychainGroup;
+
+/*!
+ Initializes with defaultKeychainGroup.
+ */
+- (nonnull instancetype)init;
```

Comment:
there's no need to declare init

Reply:
This came from the iOS keychain cache header.  Remove from both?

Reply:
updated